core/modules/cart/helpers/createMethodsData.ts

```diff
@@ -0,0 +1,44 @@
+import ShippingDetails from '@vue-storefront/core/modules/checkout/types/ShippingDetails'
+import { ShippingMethodsData } from '@vue-storefront/core/modules/cart/types/ShippingMethodsData'
+
+const getShipping = (shippingMethods: any[]) => {
+  const shipping = shippingMethods && Array.isArray(shippingMethods) ? shippingMethods.find(item => item.default && !item.offline) : null
+
+  if (!shipping && shippingMethods && shippingMethods.length > 0) {
+    return shippingMethods.find(item => !item.offline)
+  }
+
+  return shipping
+}
+
+const getPayments = (paymentMethods: any[]) => {
+  const payment = paymentMethods && Array.isArray(paymentMethods) ? paymentMethods.find(item => item.default) : null
```

Comment:
```suggestion
  if (!paymentMethods || !paymentMethods.length) return
  return paymentMethods.find(item => item.default) || paymentMethods[0]
```
and rest of it can be removed